docs/recent-releases/go-ipfs-0-5/README.md

```diff
@@ -6,27 +6,35 @@ title: Overview
 
 ![The Go-IPFS logo.](./images/go-ipfs-logo.png)
 
-It's been a while since the last Go-IPFS release, and with good reason. We've packed an incredible amount of bug fixes, improvements, and features into this build; it's the biggest Go-IPFS release since the project first started!
+It's been a while since the last Go-IPFS release. We've packed an incredible amount of bug fixes, improvements, and features into this build; it's the biggest Go-IPFS release since the project first started!
 
 ## Features
 
-There are plenty of features packed into this release. The DHS implementation is vastly improved, we refactored Bitswap to improve speed and reliability, the subdomain gateway is much easier to use, Badger has gained more support and is sufficiently stable enough for daily use, and the RSA minimum key size is now 2048. These are just some of the features available in this release. [Take a closer look →](features)
+There are plenty of features packed into this release. The DHT implementation is vastly improved, we refactored Bitswap to improve speed and reliability, go-ipfs now has built-in support for subdomain-based gateways, and Badger has gained more support and is sufficiently stable enough for daily use. These are just some of the features available in this release. [Take a closer look →](features)
```

Comment:
* subdomain gateways: this is new. The concept has existed, but go-ipfs has never had support for them.
* RSA 2048: this isn't really a feature.